src/functionalTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveRegionDetailsFunctionalTest.java

```diff
@@ -22,40 +27,107 @@
 
     public static final String mapKey = "LrdApiController.retrieveRegionDetails";
 
+    List<LrdRegionResponse> eexpectedListAll = new ArrayList<>(Arrays.asList(
```

Comment:
Please correct the variable name. An extra e is there.

Reply:
Try create Immutable list in test case